src/components/pages/geographyOriginalPage.tsx

```diff
@@ -217,6 +221,37 @@ export const GeographyOriginalPage = ({ geography, summary, targets, theme, them
     }
   };
 
+  /**
+   * Vespa search results
+   */
+  const [currentVespaSearchSelectedCategory, setCurrentVespaSearchSelectedCategory] = useState(themeConfig.categories.options[0].label);
```

Comment:
`themeConfig.categories` isn't set on all apps - on MCF and CCC where there is only one category ("All"), this property is omitted. I'd suggest referencing `documentCategories` and in particular `defaultDocumentCategory` when initialising this state variable.

As a rule of thumb, `themeConfig.categories` is for rendering the category search filters, whereas `themeConfig.documentCategories` is for showing documents by groups. I wish these were just one variable but they're not. 🤷‍♀️